model/instance/lifecycle/create.go

```diff
@@ -267,6 +268,10 @@ func buildSettings(opts *Options) *couchdb.JSONDoc {
 
 // Triggers returns the list of the triggers to add when an instance is created
 func Triggers(db prefixer.Prefixer) []job.TriggerInfos {
+	// Init metadata
+	md := metadata.New()
+	md.DocTypeVersion = job.DocTypeVersionTrigger
```

Comment:
I don't see why we are initializing the metadata here, as we are also doing the same thing in `NewTrigger`